ipfix/marshal.go

```diff
@@ -33,7 +33,7 @@ import (
 var errUknownMarshalDataType = errors.New("unknown data type to marshal")
 
 // JSONMarshal encodes IPFIX message
-func (m *Message) JSONMarshal(b *bytes.Buffer) ([]byte, error) {
+func (m *Message) JSONMarshal(b *bytes.Buffer, dataset_index int) ([]byte, error) {
```

Comment:
Nit (not sure what the convention in this repo is): https://golang.org/doc/effective_go.html#mixed-caps